src/PointwiseFunctions/GeneralRelativity/Christoffel.hpp

```diff
@@ -38,4 +45,68 @@ template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
 tnsr::abb<DataType, SpatialDim, Frame, Index> christoffel_first_kind(
     const tnsr::abb<DataType, SpatialDim, Frame, Index>& d_metric) noexcept;
 // @}
-} // namespace gr
+namespace Tags {
+/// Compute item for spatial Christoffel symbols of the first kind
+/// \f$\Gamma_{ijk}\f$ computed from the first derivative of the
+/// spatial metric.
+///
+/// Can be retrieved using `gr::Tags::SpacetimeChristoffelFirstKind`
```

Comment:
This tag doesn't seem right. Should be `gr::Tags::SpatialChristoffelFirstKind`, presumably.